nixos/modules/services/networking/monero.nix

```diff
@@ -12,6 +12,15 @@ let
 
   login = (cfg.rpc.user != null && cfg.rpc.password != null);
 
+  startScriptContents = with cfg; ''
+    #!/bin/sh
+    cat ${configFile} > ${dataDir}/monerod.conf
+    ${lib.optionalString (extraConfigFile != "") "cat ${extraConfigFile} >> ${dataDir}/monerod.conf"}
+    ${lib.getExe' pkgs.monero-cli "monerod"} --config-file=${configFile} --non-interactive
```

Comment:
I think configFile should be replaced by ${dataDir}/monerod.conf"

Reply:
You are absolutely correct.